packages/styling/components/progress-indicator.scss

```diff
@@ -0,0 +1,54 @@
+@import "../variables";
```

Comment:
Deze file moet opgeknipt worden:

* Inhoud onderbrengen in mixins in `@dso-toolkit/sources`
* In Web Component op `:host` de _root() selector toepassen
* In CSS op de huidige `.dso-progress-indicator` selector toepassen